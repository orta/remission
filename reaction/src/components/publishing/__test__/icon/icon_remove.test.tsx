import * as React from "react"
import * as renderer from "react-test-renderer"
import IconRemove from "../../icon/remove"

it("renders properly", () => {
  const icon = renderer.create(<IconRemove />).toJSON()
  expect(icon).toMatchSnapshot()
})

it("renders with fill props", () => {
  const icon = renderer.create(<IconRemove fill="red" />).toJSON()
  expect(icon).toMatchSnapshot()
})
