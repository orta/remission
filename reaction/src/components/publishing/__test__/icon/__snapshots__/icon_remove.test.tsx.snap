// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly 1`] = `
<svg
  className="remove"
  enableBackground="new 0 0 1000 1000"
  viewBox="0 0 1000 1000"
  xmlns="http://www.w3.org/2000/svg"
>
  <circle
    cx="500.937"
    cy="489.704"
    fill="#000"
    r="468.769"
  />
  <polygon
    fill="#FFFFFF"
    points="485.251,550.605 374.762,661.094 308.487,594.819 418.976,484.33 308.487,373.84 374.762,307.565 485.251,418.055 595.741,307.565 662.016,373.84 551.526,484.33 662.016,594.819 595.741,661.094 "
  />
</svg>
`;

exports[`renders with fill props 1`] = `
<svg
  className="remove"
  enableBackground="new 0 0 1000 1000"
  viewBox="0 0 1000 1000"
  xmlns="http://www.w3.org/2000/svg"
>
  <circle
    cx="500.937"
    cy="489.704"
    fill="red"
    r="468.769"
  />
  <polygon
    fill="#FFFFFF"
    points="485.251,550.605 374.762,661.094 308.487,594.819 418.976,484.33 308.487,373.84 374.762,307.565 485.251,418.055 595.741,307.565 662.016,373.84 551.526,484.33 662.016,594.819 595.741,661.094 "
  />
</svg>
`;
